### ISSUE TITLE
remote_user required for writable task?

### ISSUE THREAD

**Issue by gharlan** [deployphp/deployer#2739]
- Deployer version: 7.0-rc1
- Deployment OS: macOS

I use acl mode for writable task with writable_use_sudo enabled.

I get this error:

<img width="692" alt="Bildschirmfoto 2021-11-08 um 20 46 03" src="https://user-images.githubusercontent.com/330436/140808056-55e84e87-d35d-4ea0-91eb-5facc7d2a3cb.png">

**Comment by gharlan** [comment]
Had to set `sudo_pass`.

**Comment by antonmedv** [comment]
Can you explain a little bit?

**Comment by gharlan** [comment]
I'm not sure what happened. I saw the message above. Then I found out, that error is coming from:
https://github.com/deployphp/deployer/blob/a4801b6090c6b8f27d672e0128db074d16c6f845/src/functions.php#L389

Then I added `set('sudo_pass', '')` (at the moment I don't need a password for sudo).
After that I got this error:
> setfacl: var/log/prod-2021-11-02.log: Only directories can have default ACLs

Then I changed writable dirs to just `var` (instead of multiple subdirs `var/log` etc.).
Then the error disappeared (but I don't understand why).

Then I removed `sudo_pass` config, and it is still working.
